### PR TITLE
Adding recipe for the Python package "passwordless"

### PR DESCRIPTION
**Pull request by antonlydell**
The official Bitwarden Passwordless.dev Python library, for Python 3+.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by antonlydell** [comment]
@conda-forge/python : This PR to adds the official "Bitwarden Passwordless.dev Python library, for Python 3+ to" conda-forge. Please review my PR :)

**Comment by antonlydell** [comment]
I am willing to take on the maintainer role of this recipe.
Pinged the wrong team in the last comment: @conda-forge/help-python

File: recipes/passwordless/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  summary: 'The official Bitwarden Passwordless.dev Python library, for Python 3+.'
+  description: |
+    Your customers, users, and workforce want smooth and secure login experiences,
+    free from traditional passwords. Meet Bitwarden Passwordless.dev, the easiest
+    way to build passkeys.
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  doc_url: https://docs.passwordless.dev/guide
+  dev_url: https://docs.passwordless.dev/guide
```
Comment (synapticarbors) [inline recipes/passwordless/meta.yaml:47]: Maybe change the `dev_url` to https://github.com/bitwarden/passwordless-python

Comment (antonlydell) [reply recipes/passwordless/meta.yaml]: @synapticarbors : Good idea! I have changed the dev_url.

Comment (antonlydell) [reply recipes/passwordless/meta.yaml]: @conda-forge/help-python : Ready for a new review.

**Pull request merged**